src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends Component<{}, State> {
+  state:Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.keyPressed);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.keyPressed);
+  }
+
+  keyPressed = (event: KeyboardEvent) => {
```

Comment:
```suggestion
  handleKeyPressed = (event: KeyboardEvent) => {
```
![image](https://user-images.githubusercontent.com/96409563/236297538-bd95878c-49bc-497b-a7b6-29b3e7f2b55c.png)